packages/zwave-js/src/lib/controller/Inclusion.ts

```diff
@@ -219,6 +237,10 @@ export interface PlannedProvisioningEntry {
 	/** The device specific key (DSK) in the form aaaaa-bbbbb-ccccc-ddddd-eeeee-fffff-11111-22222 */
 	dsk: string;
 
+	/** The device must be included via ZWave Long Range */
+	// BUGBUG: by adding this here we probably broke the API used by JSUI, etc...
```

Comment:
Yeah, I probably did. Maybe this should be optional/undefined so it's backwards compatible?